cmd/agent/app/diagnose.go

```diff
@@ -39,13 +39,17 @@ var (
 		Hidden: true,
 		RunE:   doDiagnoseDatadogConnectivity,
 	}
+
+	noTrace bool
 )
 
 func init() {
 
 	diagnoseCommand.AddCommand(diagnoseMetadataAvailabilityCommand)
 	diagnoseCommand.AddCommand(diagnoseDatadogConnectivityCommand)
 
+	diagnoseDatadogConnectivityCommand.PersistentFlags().BoolVarP(&noTrace, "no-trace", "", false, "mute extra information about connection establishment, DNS lookup and TLS handshake")
```

Comment:
Instead of using `no-trace` what do you think about using `trace`? I think it is a little harder to reason about a negative  flag.

Reply:
Tracing is enable by default so users will use `--no-trace` only to disable tracing.

So if they want tracing they can use `diagnose datadog-connectivity` and if they don't want it they can use `diagnose datadog-connectivity --no-trace`